docs/source/1-fundamental-group/quest-2.rst

```diff
@@ -46,17 +46,17 @@ the higher homotopy groups of ``S¹`` might be.
 Whilst the data of the fundamental group ``π₁ S¹`` is captured
 in ``loopSpace S¹ base``,
 the data of ``π₂ S¹`` would be captured in
-``loopSpace (loopSpace S¹ base) refl``;
-loops in ``loopSpace S¹ base`` based at ``refl``.
-Points in the second loop space are paths ``h : refl ≡ refl``,
+``loopSpace (loopSpace S¹ base) Refl``;
+loops in ``loopSpace S¹ base`` based at ``Refl``.
+Points in the second loop space are paths ``h : Refl ≡ Refl``,
 i.e. ``h`` would be a homotopy from the constant path to itself.
 
 .. insert picture
 
-The second loop space contains an obvious point ``refl : refl ≡ refl``
+The second loop space contains an obvious point ``refl : Refl ≡ Refl``
 (this is of course not the same "refl" as the one before),
 and we could define the next loop space to be loops in
-``loopSpace (loopSpace S¹ base) refl`` based at (the new) ``refl``.
+``loopSpace (loopSpace S¹ base) Refl`` based at (the new) ``refl``.
```

Comment:
I'm having trouble understanding this bit of the explanation. In my mind, it feels like it should be `loopSpace (loopSpace S¹ base) Refl` (where `Refl` is the trivial path in `loopSpace S¹ base`) and `refl` is an element of `loopSpace (loopSpace S¹ base) Refl`. In other words, I think the new loopspace should be based at the old `Refl` instead of the new `refl`.

Reply:
Hi 0Art0, I'm so sorry it took me so long to respond. For some reason I wasn't getting notifications for this. So a loop space of a space `X` is based at a point `x : X` in the original space. So you are 100% correct and I was being dumb. Thanks for pointing it out :)

Reply:
ah but for consistency with the solutions I think it's best if we keep to `refl` instead of `Refl`, since nobody really defines instances of `Refl` for each type. I only made `Refl` in the first quest to give an example of how `refl` could be made.